jenkins/spark-tests.sh

```diff
@@ -284,6 +300,7 @@ export -f get_tests_by_tags
 # - DEFAULT: all tests except cudf_udf tests
 # - CUDF_UDF_ONLY: cudf_udf tests only, requires extra conda cudf-py lib
 # - ICEBERG_ONLY: iceberg tests only
+# - AVRO_ONLY: avro tests only (with --packages option instead of --jars)
```

Comment:
we should add documentation to README about this option and mention using --packages since if you don't have internet access it won't work

Reply:
https://github.com/NVIDIA/spark-rapids/tree/branch-22.10/integration_tests#enabling-apache-iceberg-tests

Reply:
looks like enabling avro test was quite different with iceberg or delta lake. Not sure whether should provide some consistency and then document it in same manner
- avro: https://github.com/NVIDIA/spark-rapids/blob/branch-22.10/integration_tests/run_pyspark_from_build.sh#L81-L85
- iceberg or delta lake: https://github.com/NVIDIA/spark-rapids/blob/branch-22.10/integration_tests/conftest.py#L45-L50

Reply:
yeah, I think this should be a new issue to bring avro test consistent w/ other ones. filed an issue to track this https://github.com/NVIDIA/spark-rapids/issues/6532